### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by pelzolga123**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,12 +10,67 @@
...
+<ul class='menu-bar'>
+  <li class='menu-bar-li'><div><i class="fab fa-apple"></i></div></li>
+  <li class='menu-bar-li'><a>Store</a></li>
+  <li class='menu-bar-li'><a>Mac</a></li>
+  <li class='menu-bar-li'><a>iPod</a></li>
+  <li class='menu-bar-li'><a>iPhone</a></li>
+  <li class='menu-bar-li'><a>iPad</a></li>
+  <li class='menu-bar-li'><a>iTunes</a></li>
+  <li class='menu-bar-li'><a>Support</a></li>
+  <li class='search-menu'><div class='search-div'><i class="fas fa-search"></i><input /></div></li>
```
Comment (bellom) [inline index.html]: The search field should be an <input> HTML tag with the focus effect enabled

File: index.html [hunk 2]
```diff
@@ -10,12 +10,67 @@
...
+   <div class='image-div'>
+     <a href="#" class='image-a'>
+        <img src='./img/third_img.png' alt=''/>
+     </a>
       
+   </div>
+   
+  </div>
+</div>
+<div class='iphone-models-div'>
```
Comment (bellom) [inline index.html]: This div should be move into the footer, which will be the top part of the footer.

File: index.html (hunk 1 above)
Comment (bellom) [inline index.html:26]: The search field should be an HTML tag with the focus effect enabled

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @pelzolga123, 
***You work is awesome, just few things left for you to do and get your :+1:*** 
Here is the checklist for your project.
- To consider the project 'complete' it must satisfy the following points:
## Search field
-  [ ] The search field should be an `<input>` HTML tag with the *focus* effect enabled
## Footer 
* [x] It should be wrapped by `<footer>` tags
* [ ] The top part is made of four `<a>` tags with `` tags inside it
* [x] The `<a>` tags should use an image as a background
* [x] The bottom part should include links like copyright, terms of use, and privacy policy

_After fixing the _**unchecked**_ issues please send another request to re-review this project._

**Review by bellom — APPROVED** [review]
Hi @pelzolga123,
## Congratulation :+1: 
Great job !! Go ahead and merge to master.

**Comment by pelzolga123** [comment]
Thank you!

**Pull request merged**